useScroll/HoverToScroll.js

```diff
@@ -0,0 +1,196 @@
+import {is} from '@enact/core/keymap';
+import {clamp} from '@enact/core/util';
+import Spotlight, {getDirection} from '@enact/spotlight';
+import {getLastPointerPosition} from '@enact/spotlight/src/pointer';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import {useCallback, useLayoutEffect, useRef} from 'react';
+
+import {getLastInputType} from '../ThemeDecorator';
+
+import css from './HoverToScroll.module.less';
+
+const nop = () => {};
+const propertyNames = (direction) => {
+	return direction === 'vertical' ? {
+		axis: 'y',
+		client: 'clientHeight',
+		maxPosition: 'maxTop',
+		scrollPosition: 'scrollTop'
+	} : {
+		axis: 'x',
+		client: 'clientWidth',
+		maxPosition: 'maxLeft',
+		scrollPosition: 'scrollLeft'
+	};
+};
+const hoverToScrollMultiplier = 0.04;
+const directionToFocus = {
+	horizontal: {
+		before: 'right',
+		after: 'left'
+	},
+	vertical: {
+		before: 'down',
+		after: 'up'
+	}
+};
+
+/**
+ * A hover area to scroll for a single direction
+ *
+ * @class HoverToScrollBase
+ * @memberof sandstone/useScroll/HoverToScroll
```

Comment:
Did you mean `sandstone/useScroll.HoverToScroll`?

Reply:
Right. I will fix this.